content/contributing/helping-out/how-you-can-help.md

```diff
@@ -59,8 +59,6 @@ If you already know what you want to work on, all you have to do is make your ch
 
 {% callout() %}
 When working on the engine's code, it's a good idea to introduce yourself in the `#engine-dev` channel on [Discord] and tell people about your plans. Communicating your progress early and often can help you avoid avoid headaches and disagreements during code review.
-
-[Discord]: https://discord.gg/bevy
```

Comment:
This is actually required because the content within `callout()` is rendered within a different context.

![image](https://github.com/user-attachments/assets/41abaaf9-6a91-4fc7-9c02-9a14438ba23f)